auth-tokens/src/main/java/com/palantir/tokens/auth/BearerToken.java

```diff
@@ -34,7 +33,7 @@
 @ImmutablesStyle
 public abstract class BearerToken {
 
-    private static final Logger log = LoggerFactory.getLogger(BearerToken.class);
+    static { LoggerFactory.getLogger(BearerToken.class); }
```

Comment:
we should special case logger subtypes in our error-prone refactor.

Reply:
Should we just ignore them or remove the line altogether?
PR for the former suggestion which doesn't seem so bad: https://github.com/palantir/gradle-baseline/pull/1146

Reply:
I don't have a strong preference either way, I think our checkstyle rules complain if loggers are unused though.